### ISSUE TITLE
docker ps data issues

### ISSUE THREAD

**Issue by hickeng** [vmware/vic#1368]
docker ps output:
```
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
da3dc0f41403        busybox             "sh"                11 weeks ago        Stopped                                 leepy_lalande
91e80e811118        busybox             "sh"                11 weeks ago        Stopped                                 ovial_rosalind
4335f5876269        busybox             "sh"                11 weeks ago        Stopped                                 onely_feynman
aeb59497ca6b        busybox             "top"               11 weeks ago        Stopped                                 istracted_visvesvaraya
6302afde43f9        busybox             "top"               11 weeks ago        Stopped                                 wesome_mcnulty
d7db62e90542        busybox             "top"               11 weeks ago        Stopped                                 ad_northcutt
20e5a5524467        busybox             "/bin/ash"          11 weeks ago        Stopped                                 etermined_austin
```

There are two major issues with this:
1. 11 weeks ago - I think this is likely the image creation time, not the container creation time.
2. the name is missing it's first character meaning it's not usable without guesswork

**Comment by hickeng** [comment]
@cgtexmex can you take a quick look at this and give an estimate of time to fix for each of the issues.

**Comment by cgtexmex** [comment]
@hickeng What steps did you take to create the data?  Did you just create?  or create and stop?

**Comment by cgtexmex** [comment]
I am able to recreate the issue around missing the first character of the friendly name, but I am unable to recreate the issue of the skewed time.

I attempted to recreate the incorrect created time with the following combos:

- create a new container without starting -- displayed correct created time with ps -a
- create a new container and start with a loop argument -- displayed correct time with ps
- create a new container and start with a command that will end -- displayed correct time with ps -a

I'll determine the issue on the name and provide an estimate shortly.

**Comment by cgtexmex** [comment]
FWIW I'm having coder's remorse on that PR #1370  -- might be best to modify the friendly name in the persona prior to display in the docker client -- thus keeping that docker "requirement" out of the stored data in the PL / vSphere layer...

**Comment by hickeng** [comment]
Does matter, it's the same regardless.
```
vagrant@devbox:~/vic$
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls create ubuntu
Unable to find image 'ubuntu:latest' locally
latest: Pulling from library/ubuntu
a3ed95caeb02: Pull complete
f069f1d21059: Pull complete
ecbeec5633cf: Pull complete
ea6f18256d63: Pull complete
54bde7b02897: Pull complete
Digest: sha256:fb6902b217019d49d0c94f0bd86ef473407d5b5182a6c5166cb84eba847b4d68
Status: Downloaded newer image for library/ubuntu:latest
b434041ecc6dcf505e0f8d40386dd2669b73384cb601607733d17ee445b10915
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
da3dc0f41403        busybox             "sh"                11 weeks ago        Stopped                                 leepy_lalande
91e80e811118        busybox             "sh"                11 weeks ago        Stopped                                 ovial_rosalind
4335f5876269        busybox             "sh"                11 weeks ago        Stopped                                 onely_feynman
aeb59497ca6b        busybox             "top"               11 weeks ago        Stopped                                 istracted_visvesvaraya
6302afde43f9        busybox             "top"               11 weeks ago        Stopped                                 wesome_mcnulty
d7db62e90542        busybox             "top"               11 weeks ago        Stopped                                 ad_northcutt
20e5a5524467        busybox             "/bin/ash"          11 weeks ago        Stopped                                 etermined_austin
b434041ecc6d        ubuntu              "/bin/bash"         11 weeks ago        Stopped                                 etermined_tesla
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls start b434041ecc6dcf505e0f8d40386dd2669b73384cb601607733d17ee445b10915
b434041ecc6dcf505e0f8d40386dd2669b73384cb601607733d17ee445b10915
vagrant@devbox:~/vic$ docker -H 192.168.78.245:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
da3dc0f41403        busybox             "sh"                11 weeks ago        Stopped                                 leepy_lalande
91e80e811118        busybox             "sh"                11 weeks ago        Stopped                                 ovial_rosalind
4335f5876269        busybox             "sh"                11 weeks ago        Stopped                                 onely_feynman
aeb59497ca6b        busybox             "top"               11 weeks ago        Stopped                                 istracted_visvesvaraya
6302afde43f9        busybox             "top"               11 weeks ago        Stopped                                 wesome_mcnulty
d7db62e90542        busybox             "top"               11 weeks ago        Stopped                                 ad_northcutt
20e5a5524467        busybox             "/bin/ash"          11 weeks ago        Stopped                                 etermined_austin
b434041ecc6d        ubuntu              "/bin/bash"         11 weeks ago        Running                                 etermined_tesla
vagrant@devbox:~/vic$
```

**Comment by cgtexmex** [comment]
There is only one issue here -- the friendly name losing the first character which is being addressed by PR #1370 

The other create time issue was due to a configuration issue -- specifically the time on the appliance server was wrong.
